### ISSUE TITLE
[BUG] `ToplevelSize::bounds` passes two null pointers

### ISSUE THREAD

**Issue by mahkoh** [gtk-rs/gtk4-rs#579]
https://github.com/gtk-rs/gtk4-rs/blob/533152fd45b251de237021af2d10a74e40611ce1/gdk4/src/toplevel_size.rs#L15-L23

Instead it should pass pointers to stack allocated ints.

**Comment by sdroege** [comment]
Indeed, thanks for reporting. Do you want to provide a PR for this?
